dev/preview/install-k3s-kubeconfig.sh

```diff
@@ -61,7 +69,4 @@ KUBECONFIG="${K3S_KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
 log "Overwriting ${KUBECONFIG_PATH}"
 mv "${MERGED_KUBECONFIG_PATH}" "${KUBECONFIG_PATH}"
 
-log "Cleaning up temporary K3S kubeconfig"
```

Comment:
Keep the k3s config, as we use that script also from the build job. Will refactor eventually.